client/scripts/controllers/server/threads.ts

```diff
@@ -29,6 +29,20 @@ export const modelFromServer = (thread) => {
     ? thread.OffchainAttachments.map((a) => new OffchainAttachment(a.url, a.description))
     : [];
 
+  const versionHistory = thread.version_history.map((v) => {
+    let history;
+    try {
+      history = JSON.parse(v || '{}');
+      history.author = history.author
+        ? JSON.parse(history.author)
+        : null;
+      history.timestamp = moment(history.timestamp);
+    } catch (e) {
+      console.log(e);
+    }
+    return history;
+  });
+
```

Comment:
Trying to understand what is happening here

Why is every entry of version_history being overwritten with a new `timestamp` and `author`?

Reply:
(See above re: comments, with the additional parsing of authors)